karrot/groups/models.py

```diff
@@ -62,9 +54,11 @@ def annotate_yesterdays_member_count(self):
             )
         )
 
+# to avoid circular imports, managers has to be imported after GroupQuerySet
```

Comment:
For consistency with other modules you should move the manager in here, right above or below GroupQuerySet (have a look at other custom managers in Karrot)